src/Geopilot.Frontend/cypress/e2e/helpers/promptHelpers.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Checks if a prompt is visible.
+ * @param {boolean} visible The expected visibility state.
+ */
+export const isPromptVisible = (visible = true) => {
```

Comment:
PP: shown verwenden, da 
```suggestion
/**
 * Checks if a prompt is shown.
 * @param {boolean} shown The expected render state.
 */
export const isPromptShown = (visible = true) => {
```

Reply:
Der Name bezieht sich auf den Visibility state: `should("be.visible")`, deshalb ist er für mich passend, aber könnens auch ändern, wenns für dich dann klarer ist.